pkg/security/token/token.go

```diff
@@ -54,7 +54,12 @@ func GetSentryToken(allowKubernetes bool) (token string, validator sentryv1pb.Si
 		if path == "" {
 			return "", sentryv1pb.SignCertificateRequest_UNKNOWN, errors.New("environmental variable DAPR_SENTRY_TOKEN_FILE is set with an empty value")
 		}
-		return GetSentryTokenFromFile(path)
+		token, validator, err = GetSentryTokenFromFile(path)
```

Comment:
Can you update the const comment to say that this env var enforces the kube validator when running in kube, else uses JWKs in self hosted 

https://github.com/dapr/dapr/blob/8cefe6a312867d8517a750d2dc300e10821bdf3a/pkg/security/consts/consts.go#L38